packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -179,115 +179,57 @@ export class LitNodeClientNodeJs
   ): Promise<CapacityCreditsRes> => {
-    const {
-      dAppOwnerWallet,
-      capacityTokenId,
-      delegateeAddresses,
-      uses,
-      domain,
-      expiration,
-      statement,
-    } = params;
+    // Useful log for debugging
+    if (!params.delegateeAddresses || params.delegateeAddresses.length === 0) {
+      log(
+        `[createCapacityDelegationAuthSig] No delegatee addresses provided. It means that the capability will not restrict access based on delegatee list, but it may still enforce other restrictions such as usage limits (uses) and specific NFT IDs (nft_id).`
+      );
+    }
 
     // -- This is the owner address who holds the Capacity Credits NFT token and wants to delegate its
     // usage to a list of delegatee addresses
     const dAppOwnerWalletAddress = ethers.utils.getAddress(
-      await dAppOwnerWallet.getAddress()
+      await params.dAppOwnerWallet.getAddress()
     );
 
-    // -- default configuration for siwe message unless there are arguments
-    const _domain = domain ?? 'example.com';
-    const _expiration =
-      expiration ?? new Date(Date.now() + 1000 * 60 * 7).toISOString();
-    const _statement = '' ?? statement;
-
-    // -- default configuration for recap object capability
-    const _uses = uses ?? '1';
-
     // -- if it's not ready yet, then connect
     if (!this.ready) {
       await this.connect();
     }
 
     // -- validate
-    if (!dAppOwnerWallet) {
+    if (!params.dAppOwnerWallet) {
       throw new Error('dAppOwnerWallet must exist');
     }
 
-    // -- validate dAppOwnerWallet is an ethers wallet
-    // if (!(dAppOwnerWallet instanceof ethers.Wallet || ethers.Signer)) {
-    //   throw new Error('dAppOwnerWallet must be an ethers wallet');
-    // }
-
-    // -- create LitRLIResource
-    // Note: we have other resources such as LitAccessControlConditionResource, LitPKPResource and LitActionResource)
-    // lit-ratelimitincrease://{tokenId}
-    const litResource = new LitRLIResource(capacityTokenId ?? '*');
-
-    const recapObject = await this.generateSessionCapabilityObjectWithWildcards(
-      [litResource]
-    );
-
-    const capabilities = {
-      ...(capacityTokenId ? { nft_id: [capacityTokenId] } : {}), // Conditionally include nft_id
-      ...(delegateeAddresses
-        ? {
-            delegate_to: delegateeAddresses.map((address) =>
-              address.startsWith('0x') ? address.slice(2) : address
-            ),
-          }
-        : {}),
-      uses: _uses.toString(),
-    };
-
-    recapObject.addCapabilityForResource(
-      litResource,
-      LitAbility.RateLimitIncreaseAuth,
-      capabilities
-    );
-
-    // make sure that the resource is added to the recapObject
-    const verified = recapObject.verifyCapabilitiesForResource(
-      litResource,
-      LitAbility.RateLimitIncreaseAuth
-    );
-
-    // -- validate
-    if (!verified) {
-      throw new Error('Failed to verify capabilities for resource');
-    }
```

Comment:
Why are we removing all these checks?

Reply:
we didn't - it's being moved to its own dedicated function in `siwe-helper.ts > addRecapToSiweMessage`

Reply:
Cool just gotta be sure that the particular function is being called